public/scripts/personas.js

```diff
@@ -855,6 +882,11 @@ async function selectCurrentPersona({ toastPersonaNameChange = true } = {}) {
                 lorebook: '',
                 connections: [],
                 title: '',
+                pronounSubjective: '',
+                pronounObjective: '',
+                pronounPosDet: '',
+                pronounPosPro: '',
+                pronounReflexive: '',
```

Comment:
If we are adding new fields, I'd prefer to add them structured.
So `pronoun.subjective`, etc. Similarly to how the macros are named. Works better than several root fields with the same prefix.